Dev3/Dev3/Employee/Junior.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Dev3
+{
+    /// <summary>
+    /// Class for junior type employee
+    /// </summary>
+    class Junior : Employee
+    {
+        public Junior()
+        {
+            Salary = 250;
+            Perfomance = 10;
+            Value = (double)Perfomance / Salary;
```

Comment:
is the counting of values different for all employees? Is it better to implement it once ?